cmd/ndm_daemonset/app/command/commands.go

```diff
@@ -37,7 +37,7 @@ func NewNodeDiskManager() (*cobra.Command, error) {
 	}
 
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
-	goflag.CommandLine.Parse([]string{})
+	_ := goflag.CommandLine.Parse([]string{})
```

Comment:
`_ = goflag.CommandLine.Parse([]string{})` is enough.  Since there are no new variables on the left side, we need to use only `=`, `:=` will give an error.

Reply:
Yup, realised I did that after seeing the output from travis. Corrected it as part of https://github.com/openebs/node-disk-manager/pull/325/commits/ae5dc2f4de3a5f82cf978284ce8f38d226def307